core/src/sumeragi/main_loop.rs

```diff
@@ -241,11 +241,32 @@ impl Sumeragi {
         assert_eq!(self.wsv.height(), 0);
         assert_eq!(self.wsv.latest_block_hash(), None);
 
-        let transactions: Vec<_> = genesis_network
-            .transactions
-            .into_iter()
-            .map(AcceptedTransaction::accept_genesis)
-            .collect();
+        let transactions: Vec<_> = {
+            let mut transactions: Vec<_> = genesis_network.transactions.into_iter().collect();
+
+            // Copy sumeragi config params into chain.
```

Comment:
Interesting, but doesn't it contradict `SetParameter` instruction?

Also why this change is needed in the benchmark context?

Reply:
I thought that it was a bug that changing config parameters did not affect iroha, but this change has been removed.